NOTICE.txt

```diff
@@ -1245,6 +1245,19 @@ SOFTWARE.
 
 ---
 
+## react-native-android-open-settings
+
+This product contains 'react-native-android-open-settings' by Leonardo Velasquez.
+
+Open Android settings from your React Native app
+
+* HOMEPAGE:
+  * https://github.com/levelasquez/react-native-android-open-settings
+
+* LICENSE: 
```

Comment:
Just a note that this doesn't have any license.

Reply:
@amyblais perhaps you can ask the author to add a LICENSE file (suggest MIT) :)

Reply:
Opened an issue: https://github.com/levelasquez/react-native-android-open-settings/issues/18

Reply:
@enahum The reporter hasn't responded yet and we're not allowed to use components that don't have a license. Do you know if this is the correct component?

Reply:
@crspeller This component doesn't have a license. Does this mean we should revert the feature PR that added this component from v1.24 for now?

Reply:
I would say no... just wait for them to respond if they end up responding that they’ll not allow the use of it or apply a license that is not compatible then we can consider changing the library

Reply:
This have the license set in the package.json to [ISC](https://en.wikipedia.org/wiki/ISC_license) which is a BSD like license, I think we are safe, but yes, they can add the LICENSE file to the repo to be more "explicit".

Reply:
The license is actually set in the package.json so we might be safe for this. 
In the future, this is above our decision level. This is a legal risk that needs to be brought to @coreyhulen and above before it makes it into a release.